web/gds-admin-ui/src/helpers/api/apiCore.js

```diff
@@ -167,6 +177,23 @@ class APICore {
         return axios.patch(url, formData, config);
     };
 
+    isUserAuthenticated = () => {
+        const user = this.getLoggedInUser();
+        if (!user) {
+            return false;
+        }
+
+        const decodedAccessToken = jwtDecode(user.access_token);
+        console.log(jwtDecode(user.refresh_token))
+        const currentTime = Date.now() / 1000;
+        if (decodedAccessToken.exp < currentTime) {
+            console.warn('access token expired');
+            return false;
```

Comment:
In the `reauthenticate` PR - will this function also check the refresh token?

Reply:
Yes